### PR TITLE
Vitaminvp DOM-api-task

### PR DESCRIPTION
**Pull request by Vitaminvp**
Vitaminvp DOM-api-task

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by AMashoshyna** [comment]
Hi @Vitaminvp! Please provide link to demo.

**Comment by Vitaminvp** [comment]
JS DOM task

[Demo](https://vitaminvp.github.io/kottans-frontend/task_js_dom/practical_task/index.html) `|`
[Code base](https://github.com/Vitaminvp/kottans-frontend/tree/master/task_js_dom/practical_task)

The code is submitted in a dedicated feature branch.
Please, review.

**Comment by OleksiyRudenko** [comment]
@Vitaminvp Please, split tasks into dedicated branches. We shan't be able to merge PR until both are approved. Also code review may easily become a mess as submitted tasks aren't related.

@AMashoshyna what do you think?

**Comment by Vitaminvp** [comment]
I deleted the last commit. Will it fit?

**Comment by OleksiyRudenko** [comment]
@Vitaminvp Much better. Thanks.

File: submissions/Vitaminvp/js-dom/script.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            e.preventDefault();
+            openContent(this.dataset.id);
+
+        });
+    };
+    const content = document.querySelector('.content');
+    openContent();
+    function openContent(contentId="0") {
+        const links = document.getElementsByClassName("side-menu__link");
+        for (let i = 0; i < links.length; i++) {
```
[the remark this answers is not in the document]
Comment (Vitaminvp) [reply submissions/Vitaminvp/js-dom/script.js]: Thank you for response. I've tried to fix all your remarks. Check please again.

File: submissions/Vitaminvp/js-dom/style.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    overflow: hidden;
+    width: 80%;
+    margin-left: 40px;
+}
+.content img {
+    max-width: 100%;
+}
+
+.ml250 {
+    margin-left: 250px;
```
Comment (AMashoshyna) [inline submissions/Vitaminvp/js-dom/style.css]: names like `w250` and `m1250` are kinda logical, but still rather risky names. Imagine that your layout will change and the panel width would become 300px wide. Then you would either leave an incorrect class name, or change both css & js, which is painful.
Let's change those names to names based on behavior. `navpanel-open` or just `open` maybe?

File: submissions/Vitaminvp/js-dom/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <div>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti dolorem ex excepturi, fugiat incidunt
+        laudantium maxime nisi non quia repellat? Enim et quod repellat.
+        Dolore iusto laboriosam nobis sunt veritatis.</div>
+      <img src="img/354.jpg" alt="">
+  </section>
+</main>
+  <div id="side-menu" class="side-nav">
+    <a href="#" class="toggle-mnu"><span></span></a>
+    <div>
+      <a class="side-menu__link active" href="#" data-id="0">Home <span>&#9654;</span></a>
```
Comment (AMashoshyna) [inline submissions/Vitaminvp/js-dom/index.html]: No sure about `anchor` elements in this context. Those buttons are not really links. Navigation panels are often implemented as `nav` element with unordered list inside. Take a look a this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)

**Comment by AMashoshyna** [comment]
Very good job with js issues! Take a closer look a css & html.

File: submissions/Vitaminvp/js-dom/index.html (hunk 3 above)
Comment (Vitaminvp) [reply submissions/Vitaminvp/js-dom/index.html]: Ok. Thank's. Check please again.

**Comment by AMashoshyna** [comment]
LGTM

**Pull request merged**